javascript-modal/main.js

```diff
@@ -0,0 +1,11 @@
+var $openButton = document.querySelector('.blue-button');
+var $closeButton = document.querySelector('.close-button');
+var $opensaseme = document.querySelector('.modal');
+function opensaseme() {
+  $opensaseme.className = 'modal';
+}
+function closesaseme() {
+  $opensaseme.className = 'modal-hide';
+}
```

Comment:
1. It's `sesame` not `saseme`
2. You should be using `camelCase`
3. You should give this a name that actually describes its purpose rather than a joke 😉